### ISSUE TITLE
part code table cleanup: sternal

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#7753]
### Goal
_Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

Remove confusing or duplicate part names

### Context
_Describe why this new value is necessary and existing values are not._

Sternum and sternal are overlapping and people won't find what they are looking for, in addition, sternal also exists in the part modifier table. 

Sternal is only used in four records:

https://arctos.database.museum/guid/UTEP:ES:32-143
https://arctos.database.museum/guid/UTEP:ES:60-31
https://arctos.database.museum/guid/UAM:ES:12835
https://arctos.database.museum/guid/UAM:ES:13436

For the UTEP records, I changed the parts to sternum and added the part modifier sternal, if we can change the UAM parts, the part would no longer be in use and could be removed from the code table.

### Table
_Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name

### Proposed Value
_Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

~~sternal~~

### Proposed Definition
_Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

N/A - remove from the code table

### Collection type
_Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

#### Attribute data type
_If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_

#### Attribute controlled values
_If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate._

#### Attribute units
_if numerical values should be accompanied by units, provide a link to the appropriate units table._

#### Part preservation attribute affect on "tissueness"
_if a new part preservation is requested, please add the affect it would have on "tissueness": No Influence, Allows, or Denies_

### Priority
_Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

let's not let this get used again

### Example Data
_Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

### Available for Public View
_Most data are by default publicly available. Describe any necessary access restrictions._

### Helpful Actions

- [x] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [x] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators @mvzhuang 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Jegelewicz** [comment]
> Patrick,
> 
> In an attempt to normalize our part names and remove confusing information, we are looking to remove [sternal](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#sternal) from the part name list in Arctos. Your collection is the only one using this term. Please see the Github issue [here](https://github.com/ArctosDB/arctos/issues/7753).
> 
> If you have a strong objection to this change, please comment there, otherwise let us know if we can make this change for you, or if you make the change, let us know so that we can remove the term from the code table.
> 
> Thank you!

Teresa J. Mayfield-Meyer

**Comment by Jegelewicz** [comment]
> I don’t have strong objections, but wonder why this is an issue. I read the Github text. My responses:
> 
> 1) It is a legit anatomical name (a bone found in critters like duck-billed dinosaurs).
> 2) It is NOT a sternum (which is a single midline bone in mammals; a sternal is a paired element in the shoulder girdle).
> 3) It is used in the peer reviewed literature; anyone looking for a sternal would not look for a sternum!
> 
> Anyway, those are my thoughts.
> best

Pat

**Comment by Jegelewicz** [comment]
Can we link sternal to an ontology? I do not find a definition for this bone in a Google search and wonder if we could use

girdle with condition = partial, part modifier = pectoral and sternal in part remark?

@Nicole-Ridgwell-NMMNHS @KatherineLAnderson thoughts?

**Comment by KatherineLAnderson** [comment]
> Can we link sternal to an ontology? I do not find a definition for this bone in a Google search and wonder if we could use
> 
> girdle with condition = partial, part modifier = pectoral and sternal in part remark?
> 
> @Nicole-Ridgwell-NMMNHS @KatherineLAnderson thoughts?

The sternum and rib cage (including sternal ribs) are not part of the pectoral girdle, so I don't think this is accurate.

I think sternal could be changed to sternum for the UAMES specimens. Or, they could mean sternal rib (https://www.merriam-webster.com/dictionary/sternal%20rib), in which case part=rib with sternal in remark would be appropriate.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Perhaps clarify that part of the reason for removing the term is usage - we are trying to remove terms with low usage. Also, even if the value is in remarks, it will still be found in part search.

**Comment by KatherineLAnderson** [comment]
^Sorry I missed the email response from Pat D (UAMES) above Teresa's last comment.

Obviously this is a confusing thing because I'm definitely confused about what a "sternal" even is. Can Pat provide documentation?

**Comment by dustymc** [comment]
adding to agenda, this should be a simple decision affecting only two records at the moment

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Issues meeting says we can keep, just come up with a better definition/reference. I will see what I can find.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
> It is NOT a sternum (which is a single midline bone in mammals; a sternal is a paired element in the shoulder girdle).

I haven't been able to find anything to back up this part of Pat's statement. My understanding after reading through a bunch of references is that a sternal plate is a sternum that has been ossified into paired bones (as opposed to the more common cartilaginous sternum or ossified midline bone such as in mammals and birds).

This is the most useful online reference I was able to find: https://doi.org/10.1080/02724634.2019.1700992 (available in full here: [researchgate](https://www.researchgate.net/profile/Nathan-Smith-61/publication/339053510_Sternal_elements_of_early_dinosaurs_fill_a_critical_gap_in_the_evolution_of_the_sternum_in_Avemetatarsalia_Reptilia_Archosauria/links/5e7791b94585157b9a54485e/Sternal-elements-of-early-dinosaurs-fill-a-critical-gap-in-the-evolution-of-the-sternum-in-Avemetatarsalia-Reptilia-Archosauria.pdf))

Here is a definition pulled from the abstract of that article: "An ossified sternum that consists of paired plates situated just posterior to the clavicles and scapulocoracoid"

**Comment by dustymc** [comment]
>ossified sternum


https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name#sternum

>The breastplate, a bone which is connected to and suspended by the false (ventral) ribs. In advanced birds, the sternum is keeled to allow for the attachment of large flight muscles.

"sternum" could be a lot more 'scholarly,' but I'll tentatively suggest we DO NOT have enough information to retain 'sternal' at this time; I don't see a clear disambiguation from the commonly-used part.

**Comment by dustymc** [comment]
Done.
